BREAKING.md

```diff
@@ -334,8 +335,6 @@ Additionally, please note that the `Connecting` state is being renamed to `Catch
 `ConnectionState.Connecting` is marked as deprecated, please use `ConnectionState.CatchingUp` instead.
 `ConnectionState.Connecting` will be removed in the following major release.
 
-### Remove ICodeLoader from `@fluidframework/container-definitions`
```

Comment:
This is suspicious... Did we not do what this says?  If it was reverted I would leave it in with a big bold note saying it was reverted.  Maybe strikethrough the text (I did this with bindToContext last week)

Reply:
This note was actually moved, not deleted. It was originally placed into the wrong section and was moved into the correct one in this PR. You can find this note on line 141 now